### ISSUE TITLE
SSP discontinuity at age ~6 Gyr — predict_rest_sed has non-physical step in Mg b region

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#299]
## Summary
`predict_rest_sed` for single-burst populations shows a sharp, non-physical **discontinuity at age ~6 Gyr** in the Mg b absorption region (5170–5180 Å), with ratio jumping up and plateauing.

## Diagnostic Results
Mg b pseudo-EW ratio [flux(5170-5180 Å) / flux(5100-5120 Å)] for a Gaussian-burst model (tsnorm: peak_lbt_gyr=age, width=0.05 Gyr):

```
age (Gyr)  ratio(5170-5180)/(5100-5120)
---------------------------------------------
   1.0      0.9612
   2.0      0.9344
   5.0      0.9188
   5.3      0.9165  ← smooth trend
   5.5      0.9165
   6.0      0.9447  ← sharp jump +0.0282 (~3%)
   7.0      0.9447  ← flat-lines
   8.0      0.9447
  11.0      0.9447
```

## Expected Behavior
Mg b strength increases monotonically with age (stellar population aging). The ratio should increase smoothly or stay flat, never jump discontinuously.

## Likely Cause
- SSP grid discontinuity or gap near age ~6 Gyr
- Interpolation artifact (e.g., switched SSP source, metallicity bin change, or age grid discontinuity at 6 Gyr)
- Possible issue with 'tsnorm' SFH type encountering an age boundary in the grid

## Repro Script
```python
import tengri, jax, numpy as np
ssp = tengri.load_ssp()
ages_gyr = [1.0, 2.0, 5.0, 5.3, 5.5, 6.0, 7.0, 8.0, 11.0]
for age in ages_gyr:
    m = tengri.SEDModel.build(ssp,
        sfh={'type':'tsnorm','*':tengri.FIXED,'peak_lbt_gyr':age,'width_gyr':0.05,'log_peak_sfr':1.0,'skew':0.0,'trunc':13.5},
        dust={'type':'two_component','*':tengri.FIXED,'tau_diff':0.0,'tau_bc':0.0},
        redshift=tengri.Fixed(0.05))
    p = dict(m.spec.sample(jax.random.PRNGKey(0)))
    out = m.predict_rest_sed(p)
    w = np.asarray(out.wavelength); L = np.asarray(out.sed)
    mask = (w > 5170) & (w < 5180)
    flux_mgb = float(np.mean(L[mask]))
    cont_mask = (w > 5100) & (w < 5120)
    flux_cont = float(np.mean(L[cont_mask]))
    print(f'age={age:.1f} Gyr  ratio = {flux_mgb/flux_cont:.4f}')
```

## Labels
@suchethacooray: needs investigation of SSP grid structure around age 6 Gyr boundary.
